stablehlo/reference/InterpreterOps.cpp

```diff
@@ -123,6 +123,10 @@ SmallVector<InterpreterValue> evalRunParallelOp(
 
   SmallVector<InterpreterValue> results;
   for (auto &future : futures) results.append(future.get());
+
+  while (!processGrid.outfeedEmpty())
+    for (auto outfeed : processGrid.popOutfeed()) results.push_back(outfeed);
```

Comment:
This made me realize that RunParallelOp::verify should also verify the number of results, not just the number of inputs.